src/elisp/treemacs-customization.el

```diff
@@ -32,11 +32,12 @@
            (s-trim)
            (s-lines)
            (--first
-            (->> (concat (shell-quote-argument it) " --version")
+            (if (file-exists-p it)
```

Comment:
Without an else branch this should be a `ẁhen`.

Reply:
Replaced the if statement with a when statement